frontend/style/globals.css

```diff
@@ -146,3 +143,18 @@ p {
 .maplibregl-marker {
   z-index: 10000;
 }
+
+/* # that appears before headings in content extracted from markdown */
+.anchor-link {
+  font-size: 1em;
+  float: left;
+  margin-left: -1em;
+  padding-right: .25em;
+  color: #000091;
+  opacity: 0;
+  background-image: none !important;
+}
```

Comment:
I generally avoid `em` and always prefer `rem` but it does sound like this is one of the few case where `em` is more relevant 👍

Reply:
I copied your code from data.gouv, but yeah, I agree that its better using `em` since we want to adapt to the container font-size.

Reply:
hahaha I was wondering ! And I probably copied it myself from a smart programmer somewhrere 🤣